mteb/tasks/STS/por/SickBrSTS.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskSTS import AbsTaskSTS
+
+
+class SickBrSTS(AbsTaskSTS):
+    metadata = TaskMetadata(
+        name="SICK-BR",
+        dataset={
+            "path": "eduagarcia/sick-br",
+            "revision": "0cdfb1d51ef339011c067688a3b75b82f927c097",
+        },
+        description="SICK-BR is a Portuguese inference corpus, aligned to and translated from SICK",
+        reference="https://huggingface.co/datasets/eduagarcia/sick-br",
+        type="STS",
+        category="s2s",
+        eval_splits=["validation", "test"],
+        eval_langs=["por-Latn"],
+        main_score="cosine_spearman",
+        date=None,
+        form=None,
+        domains=None,
+        task_subtypes=None,
+        license=None,
+        socioeconomic_status=None,
+        annotations_creators=None,
+        dialect=None,
+        text_creation=None,
+        bibtex_citation=None,
+        n_samples=None,
+        avg_character_length=None,
+    )
+
+    @property
+    def metadata_dict(self) -> dict[str, str]:
+        metadata_dict = super().metadata_dict
+        metadata_dict["min_score"] = 1
+        metadata_dict["max_score"] = 5
+        return metadata_dict
+
+    def dataset_transform(self):
+        self.dataset = self.dataset.rename_columns(
+            {
+                "sentence_A": "sentence1",
+                "sentence_B": "sentence2",
+                "relatedness_score": "score",
+                "pair_ID": "id",
+            }
+        )
```

Comment:
I believe subsampling would be nice here. We are currently making efforts to make the benchmark run faster and I highly doubt having 2000 examples would seriously  change scores.

Reply:
I would just do a train-test split (either with Huggingface or with sklearn) on the test set and only keep one half. Some other tasks do have code like this, if you have a hard time implementing it, I can write you some code bits tomorrow.

Reply:
Understood, thank you @x-tabdeveloping -- I went with a shuffle and a straightforward `select` after that. I'd appreciate if you could take a look.